justfile

```diff
@@ -4,8 +4,7 @@
 # Configuration
 #
 
-export RUST_BACKTRACE := "short"
-export RUSTFLAGS := env_var_or_default("RUSTFLAGS", "-D warnings -A deprecated")
```

Comment:
TIOLI: the `justfile` already [detects whether we're on github actions](https://github.com/linkerd/linkerd2-proxy/blob/45d6cf386f71cf284d238e35f5ff149d535cbdf3/justfile#L54), so we could alternatively just set this based on whether the `GITHUB_ACTIONS` env var is set or not? that way, we wouldn't have to stick the env var in every CI config. but, on the other hand, it's maybe more explicit to have all the CI workflows explicitly state that they disable warnings? i dunno...